pkg/cli/show-config.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Chainguard, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
+	"chainguard.dev/apko/pkg/build"
+)
+
+func ShowConfig() *cobra.Command {
+	var extraKeys []string
+	var extraRepos []string
+
+	cmd := &cobra.Command{
+		Use:   "show-config",
+		Short: "Show the configuration derived from loading a YAML file",
+		Long: `Show the configuration derived from loading a YAML file.
+
+The derived configuration is rendered in YAML.
+`,
+		Example: `  apko show-config <config.yaml>`,
```

Comment:
```suggestion
		Example: "apko show-config <config.yaml>",
```

Unless there's a reason for it to have spaces prefixed?

Reply:
That's how cosign did it, I copied cosign :P